plugins/node/opentelemetry-instrumentation-ioredis/package.json

```diff
@@ -72,7 +74,6 @@
   "dependencies": {
     "@opentelemetry/instrumentation": "^0.32.0",
     "@opentelemetry/semantic-conventions": "^1.0.0",
-    "@types/ioredis": "4.26.6"
```

Comment:
I think this type is exported publicly from `types.d.ts` so we still need to have it as a dependency.
Now V5 types are also exported so they should probably be added as regular dependency as well.

Reply:
Do you mean having the types for 4.x and ioredis 5 (types are in the package) as a dependency?

Reply:
Yes, since they are exported from `types.d.ts`, a typescript user that installs the instrumentation but not the redis types packages, will get a transpilation error.

You can read more in the [instrumentation guide](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/GUIDELINES.md#types-public-api)